lib/modules/ai_bot/personas/persona.rb

```diff
@@ -127,8 +150,12 @@ def all_available_commands
             all_commands << Commands::GoogleCommand
           end
 
-          allowed_commands = SiteSetting.ai_bot_enabled_chat_commands.split("|")
-          @cmds = all_commands.filter { |klass| allowed_commands.include?(klass.name) }
+          all_commands
+        end
+
+        def all_available_commands
+          return @cmds if @cmds
```

Comment:
Can't you do `@cmds ||= self.class.all_available_commands` here?

Reply:
sure will change